serpent-moltres/full-core/extract-convert.py

```diff
@@ -0,0 +1,320 @@
+#!/usr/bin/env python3
+# This file is based in $MOLTRES/python/extractSerpent2GCs.py
+import os
+import numpy as np
+import argparse
+import subprocess
+import serpentTools as sT
+
+
+def makePropertiesDir(outdir, filebase, mapFile, unimapFile, ngroups='3'):
```

Comment:
This function is 20x too long to test. It's 10x too long for a reader to follow its logic.